docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -210,7 +233,75 @@ data:
 EOF
 ----
 +
-The resources in the `ConfigMap` contain cell-specific configurations.
+
+. Create dataplane services for Nova cells to enable pre-upgrade workarounds,
+and to configure the {compute_service} services for a picked storage back end:
+
++
+[source,yaml]
+----
+for CELL in $(echo $RENAMED_CELLS); do
+  oc apply -f - <<EOF
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: nova-$CELL
+spec:
+  dataSources: <1>
+    - secretRef:
+        name: nova-$CELL-compute-config <2>
+    - secretRef:
+        name: nova-migration-ssh-key <3>
+    - configMapRef:
+        name: nova-cells-global-config
+        optional: true
```

Comment:
do we need to define this optional? is it used later to be able to delete the global-config CM?